lib/common/url.c

```diff
@@ -423,7 +423,7 @@ int h2o_url_init_with_hostport(h2o_url_t *url, h2o_mem_pool_t *pool, const h2o_u
     } else {
         url->_port = port;
         char _port[sizeof(H2O_UINT16_LONGEST_STR)];
-        int port_len = sprintf(_port, "%" PRIu16, port);
+        int port_len = sprintf(_port, "%h" PRIu16, port);
```

Comment:
What is the warning you are seeing?

`port` is `uint16_t`; and IIUC the type specifier _is_ `"%" PRIu16`.

Reply:
Was unable to reproduce the warning I got a while ago.